Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -606,6 +606,27 @@
   loadouts:
   - JesterHat
   - JesterAltHat
+  # Moffstation - Begin - Clown Clothes
+  - HarlequinHat
+
+- type: loadoutGroup
+  id: ClownMask
+  name: loadout-group-clown-mask
+  minLimit: 1
+  loadouts:
+  - ClownMaskDefault
+  - ClownMaskBlushing
+  - ClownMaskBanana
+  - ClownMaskCool
+  - HarlequinMask
+
+- type: loadoutGroup
+  id: ClownNeck
+  name: loadout-group-clown-neck
+  minLimit: 0
+  loadouts:
+  - RuffleNeck
+  # Moffstation - End - Clown Clothes
```

Comment:
Entirely new prototypes (ie. things where you do a top level `- type: ...`) need to be put into a `_Moffstation` directory.